extensions/kill-node-modules/src/kill-node-modules.tsx

```diff
@@ -0,0 +1,104 @@
+import { ActionPanel, Action, Icon, List, showToast, Toast } from "@raycast/api";
+import { useNodeModules } from "./hooks/useNodeModules";
+import { useMemo, useState, useCallback } from "react";
+import fsExtra from "fs-extra";
+import { exec } from "child_process";
+import { promisify } from "util";
+import path from "path";
+import { formatSize } from "./utils/format";
+
+const execPromise = promisify(exec);
+
+export type NodeModulesItem = {
+  id: string;
+  title: string;
+  lastModified: Date;
+  size: number;
+};
+
+export default function Command() {
+  const { items, loading, pagination } = useNodeModules();
+  const [deletedItems, setDeletedItems] = useState<Set<string>>(new Set());
+  const [deletingItems, setDeletingItems] = useState<Set<string>>(new Set());
+  const [sortWith, setSortWith] = useState<"size" | "lastModified">("lastModified");
+
+  const handleDelete = useCallback(async (id: string, path: string) => {
```

Comment:
Should we add a [confirmAlert](https://developers.raycast.com/api-reference/feedback/alert#confirmalert) with       `rememberChoice: true`